base/broadcast.jl

```diff
@@ -127,6 +127,7 @@ end
 end
 
 Base.@propagate_inbounds _broadcast_getindex(A, I) = _broadcast_getindex(containertype(A), A, I)
+Base.@propagate_inbounds _broadcast_getindex(::Type{Tuple}, A::Tuple{Any}, I) = A[1]
```

Comment:
Should `A[1]` not be `A[I]`?

Reply:
No, you can see that there is a `::Tuple{Any}` in the signature, we need this for handling cases like `(1,2,3) .+ (1,)`. This shouldn't be a problem as we want `(x,)` where `x` is a scalar to behave the same as `[x]` or `x`.

Reply:
Yes, I missed the `Any` last night :). To make this method's purpose clearer, perhaps add a comment or an adjacent general method
```julia
Base.@propagate_inbounds _broadcast_getindex(::Type{Tuple}, A::Tuple, I) = A[I]
```
though it would be redundant with
```julia
Base.@propagate_inbounds _broadcast_getindex(::Any, A, I) = A[I]
```
below?

Reply:
I have added a comment.